pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -413,4 +416,26 @@ self: super: {
     '';
   });
 
+  vCoolor-vim = super.vCoolor-vim.overrideAttrs(old: {
+    # on linux can use either Zenity or Yad.
+    propagatedBuildInputs = [ gnome3.zenity ];
+    meta = {
+      description = "code-completion for python using python-jedi";
+      license = stdenv.lib.licenses.mit;
+    };
+  });
+
+  unicode-vim = let
+    unicode-data = fetchurl {
+      url = http://www.unicode.org/Public/UNIDATA/UnicodeData.txt;
+      sha256 = "16b0jzvvzarnlxdvs2izd5ia0ipbd87md143dc6lv6xpdqcs75s9";
+    };
+    in super.unicode-vim.overrideAttrs(old: {
+
+      # redirect to /dev/null else changes terminal color
+      buildPhase = ''
+        cp ${unicode-data} autoload/unicode/UnicodeData.txt
```

Comment:
Quotes would be nice, although not necessary. Also if we're redirecting to `/dev/null`, we should at least print a status line ("Building unicode cache" or something) to make debugging easier.

Reply:
done